src/content/docs/style-guide/images/embed-videos.mdx

```diff
@@ -13,37 +13,122 @@ redirects:
+4. Click **Add > Upload**.
+5. Click **Embed &#38; Share**.
+6. Copy the video ID from the embed code so you can insert it into your video link. For example, `jsh48c18dq` from this tag:
+
+  ```
+  <script src="https://fast.wistia.com/embed/medias/<var>jsh48c18dq</var>.jsonp" async></script><script src="https://fast.wistia.com/assets/external/E-v1.js" async></script><div class="wistia_responsive_padding" style="padding:216.88% 0 0 0;position:relative;"><div class="wistia_responsive_wrapper" style="height:100%;left:0;position:absolute;top:0;width:100%;"><div class="wistia_embed wistia_async_jsh48c18dq videoFoam=true" style="height:100%;position:relative;width:100%"><div class="wistia_swatch" style="height:100%;left:0;opacity:0;overflow:hidden;position:absolute;top:0;transition:opacity 200ms;width:100%;"><img src="https://fast.wistia.com/embed/medias/jsh48c18dq/swatch" style="filter:blur(5px);height:100%;object-fit:contain;width:100%;" alt="" aria-hidden="true" onload="this.parentNode.style.opacity=1;" /></div></div></div></div>
+  ```
+7. Insert the ID into the video tag in your document. Note that you can also add properties to control the video (see [Sizing videos](#sizing)).
+
+  ```
+  <Video
+    type="wistia"
+    id=”jsh48c18dq"
+  />
+  ```
+
+## YouTube  [#youtube-options]
+
+You have two options to embed videos from YouTube: the `<iframe>` tag or our `<Video>` component.
+
+### Use iframe tag from YouTube [#use-iframe]
+
+Use the embed code that you can find on the YouTube site:
+
+1. Go to the YouTube and find the video you want to embed.
+2. Click **Share** and then click **Embed**.
+3. Copy the entire `<iframe>` tag. For example:
+
+  ```
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/<var>04JP0ky_hjI</var>" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+  ```
+4. Paste that tag into your document.
+
+Here's how it is displayed:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/04JP0ky_hjI" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+### Use our Video component [#video-comp]
+
+You can use the `Video` component in the same way you use it for videos hosted on [Wistia](#wistia-options). Copy the video ID from YouTube and insert it into the `Video` component. Note that you can also add properties to control the video (see [Sizing videos](#sizing)).
+
+Here's an example without any additional properties:
+```
+<Video
+  type="youtube"
+  id="<var>04JP0ky_hjI/var>"
+/>
+```
+Here's how it is displayed:
+
+<Video
+  type="youtube"
+  id="04JP0ky_hjI"
+/>
+
+## Sizing videos [#sizing]
+
+When you use the `<Video>` component you can add properties to control how it is displayed.
+
+### Width [#width]
+
+We recommend that you only set width and not height. Here's an example:
```

Comment:
so it is not possible to set a `height` on the video, so it is only possible to set a `width` so even if users tried to set a height it would do nothing

Reply:
@caylahamann thanks! I'll clarify this in the doc!